feature_engine/timeseries/forecasting/base_forecast_transformers.py

```diff
@@ -51,6 +42,9 @@ class BaseForecastTransformer(BaseEstimator, TransformerMixin, GetFeatureNamesOu
 
     {drop_original}
 
+    group_by_variables: str, list of str, default=None
```

Comment:
I'd call this method `group_by` to keep it similar to pandas method.

Reply:
Also, we do allow variable names to be integers, so in theory it could also take int and list of ints.

Reply:
yes, sure I am going to rename this parameter, and also allow to pass it as integer or list of integers, and use check_variables methods to check if it exists in dataframe, thank you.